docs/authentication/saml-at-clerk.mdx

```diff
@@ -1,3 +1,144 @@
+
+For example, the claim `public_metadata_country` with the value `Canada` will be saved in the user's `public_metadata` under the key `country` with the value `Canada`.
+
+Learn more about how to access the [metadata](/docs/users/metadata) from our APIs.
+
+## Activate your SAML Connection
+
+Now that everything is set up, you need to activate the SAML Connection via the Clerk Dashboard. This is so you don't expose your connection to your users while you're in the, sometimes long, process of configuring and testing it.
+
+## Authenticate with SAML
+
+Everything should be set up now for you to try out authentication via SAML. Go to your application’s Sign In page and add your email in the input field. If it matches an active SAML Connection, you will be redirected to your IdP where you will log in with your credentials.
+
+## Glossary
+| Term | Explanation | Azure AD Terminology | Google Terminology | Okta Terminology |
+| --- | --- | --- | --- | --- |
+| `idp_entity_id` | This is the unique identifier of your IdP’s application. | Azure AD Identifier | Entity ID | Identity Provider Issuer |
+| `idp_sso_url` | This is your IdP’s URL that we’ll redirect your users to so that they authenticate. | Login URL | SSO URL | Identity Provider Single Sign-On URL |
+| `idp_certificate` | This is the certificate needed for Clerk to securely connect to your IdP. | Certificate (Base64) | Certificate | x.509 Certificate |
+| `sp_entity_id` | This is a unique identifier for your SAML connection that your IdP application needs. | Identifier (Entity ID) | Entity ID (Under “Service Provider Details”) | Audience URI (SP Entity ID) |
+| `acs_url` | This is your application’s URL that your IdP will redirect your users back to after they have authenticated in your IdP. | Reply URL (Assertion Consumer Service URL) | ACS URL | Single sign-on URL |
+| SAML | This is the protocol that we are using to connect to your Identity Provider. Other protocols are OIDC, OAuth 2.0. | | | |
+| SAML Connection | This is the connection that you create in Clerk to hold the integration configuration. | | | |		
+| Identity Provider | This is the service you are using to store your users’ identity. Azure AD and Okta are two popular providers. | | | |	
+| IdP | Abbreviation for “Identity Provider” | | | |		
+| IdP Application | This is an application from the IdP that connects to | | | |		
+| Service Provider | This is the service that requests and receives the identity from the IdP. | | | |	
+| SP | Abbreviation for “Service Provider”. | | | |	
+
+## FAQ
+
+**I’ve enabled other strategies but they don’t work.**
+
+It is expected that once Enterprise SSO is enabled in an organization, there should be no other authentication methods applicable. This is in line with an organization’s intent to manage their users’ identity from one place.
```

Comment:
"It is expected that once Enterprise SSO is enabled in an organization"

I believe this should read "It is expected that once Enterprise SSO is enabled in an application"

I'm not aware of a way to tie them to an Organization only.

Reply:
I should add that with the upcoming auto-join work with Orgs that is tired to domains, this or other areas of the SAML doc might need updating. I'm not really sure how that work will affect this.